sdcm/tester.py

```diff
@@ -1812,17 +1812,17 @@ def _get_common_email_data(self) -> dict:
         start_time = format_timestamp(self.start_time)
         config_file_name = ";".join(os.path.splitext(os.path.basename(cfg))[0] for cfg in self.params["config_files"])
         critical = self.get_test_failing_events()
-
+        test_status = "FAILED" if critical else "SUCCESS"
         return {"build_url": os.environ.get("BUILD_URL"),
                 "end_time": format_timestamp(time.time()),
                 "events_summary": get_logger_event_summary(),
                 "last_events": EVENTS_PROCESSES['EVENTS_FILE_LOOGER'].get_events_by_category(100),
                 "nodes": [],
                 "start_time": start_time,
-                "subject": f"Result {os.environ.get('JOB_NAME') or config_file_name}: {start_time}",
+                "subject": f"{test_status}: {os.environ.get('JOB_NAME') or config_file_name}: {start_time}",
```

Comment:
@roydahan  we added a status to the subject since it is easier to see the status of the test on the inbox